environment.yml

```diff
@@ -18,5 +18,6 @@ dependencies:
   - distro=1.5.0
   - paramiko=2.7.1
   - pip
+  - pytest
```

Comment:
Version?

Reply:
pytest more or less doesn't need a version since its a dev dependency. I can change the travis script so It is not included in this file but I don't think a version is required here. If we find that a specific version has a feature or behaves in a desirable way then we can add that but for now its fine.